tour_model_eval/viz_bins_clusters_above_cutoff.ipynb

```diff
@@ -3,7 +3,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "contemporary-bennett",
+   "id": "proud-recruitment",
```

Comment:
nit: why are all these IDs different?

Reply:
I don't know. Is it due to change of network? checkpoints in Jupiter notebook?